Latest applet and apdu java files.

Nomit1993 opened the pull request.

mgrabovsky commented:
There seems to be a big mismatch between the latest revision (in this repository) and your proposed changes. For example, I've changed the names of some classes and added a few comments here and there. Can you please update your code to match the current state?

Nomit1993 commented:
Ok. I will do that.

On Wed, Apr 22, 2020, 12:20 PM Matěj G. <notifications@github.com> wrote:

> There seems to be a big mismatch between the latest revision (in this
> repository) and your proposed changes. For example, I've changed the names
> of some classes and added a few comments here and there. Can you please
> update your code to match the current state?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mgrabovsky/pv204-spring2020-project/pull/8#issuecomment-617689203>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANUMPUNHWUBHTMQSUVN4CU3RN3AGRANCNFSM4MOA5HRQ>
> .
>

Nomit1993 commented:
Hey Matej, so i have tried to update the codes in my forked repo. Kindly
have a look and update it to the main project.
One thing that i did not implement was setting up PIN as i am taking pre
set PIN of 1234 for comparing. Kindly fit in that part again.
Thank You.

On Wed, Apr 22, 2020 at 12:28 PM N S <sharma.nomit@gmail.com> wrote:

> Ok. I will do that.
>
> On Wed, Apr 22, 2020, 12:20 PM Matěj G. <notifications@github.com> wrote:
>
>> There seems to be a big mismatch between the latest revision (in this
>> repository) and your proposed changes. For example, I've changed the names
>> of some classes and added a few comments here and there. Can you please
>> update your code to match the current state?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mgrabovsky/pv204-spring2020-project/pull/8#issuecomment-617689203>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ANUMPUNHWUBHTMQSUVN4CU3RN3AGRANCNFSM4MOA5HRQ>
>> .
>>
>
